layouts/community/ortho_4x12/jotix/config.h

```diff
@@ -1 +1,17 @@
-#define TAPPING_TOGGLE 2
+/*
+ * Used to create a keymap using only KC_ prefixed keys
+ */ 
+#define LAYOUT_kc_ortho_4x12( \
```

Comment:
Layout macros that automatically prepend `KC_` are discouraged, as they do not play well with the Configurator, and make some of the more advanced keycodes such as `MO()` and `LT()` harder to use, since you need to manually define aliases beginning with `KC_`.